packages/react-portal/src/usePortalMountNode.ts

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { useIsomorphicLayoutEffect, isSSR } from '@fluentui/react-utilities';
+import { useThemeClassName, useFluent } from '@fluentui/react-shared-contexts';
+
+export type UsePortalElementOptions = {
+  /**
+   * The document where the portal element will be created
+   *
+   * @defaultValue document from Fluent context
+   */
+  targetDocument?: Document | undefined;
+  /**
+   * HTML dir attribute
+   *
+   * @defaultVaue dir from Fluent context
+   */
+  dir?: React.HTMLAttributes<HTMLElement>['dir'];
+  /**
+   * CSS class to apply
+   */
+  className?: string;
+
+  /**
+   * Since hooks cannot be called conditionally use this flag to disable creating the node
+   */
+  disable?: boolean;
```

Comment:
```suggestion
  disabled?: boolean;
```

IMO we should be consistent. In native attributes it's `disabled`

Reply:
should I change the naming ? it's not actually the DOM  `disabled` attribute. It's just that it's not possible to call hooks conditionally, so I need a flag to stop the creation of the DOM element unnecessarily

Reply:
> should I change the naming ? it's not actually the DOM disabled attribute.

I suggest to go with `d` ending for this case.